src/objects/Testing_Configuration__c.object

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm still trying to get through this PR, but why would there be an object called "Testing Configuration"? Is "Testing" what this is actually used for?
Also, a description is required for this object even if it is protected. I suppose that would help with my previous question too.